roles/anaconda/templates/run_conda_init.sh.j2

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+#
+echo "Adding asreview command function that launches virtualenv"
```

Comment:
What does this message mean?

Reply:
It is technically just a way of logging the different entries inside the runonce role.